spring-batch-core/src/main/java/org/springframework/batch/core/step/tasklet/SystemCommandTasklet.java

```diff
@@ -64,6 +64,8 @@
 
 	protected static final Log logger = LogFactory.getLog(SystemCommandTasklet.class);
 
+	private static CommandRunner commandRunner = new JvmCommandRunner();
```

Comment:
I would like to write some tests that use a mocked `CommandRunner`. Would it make sense to add them in `SystemCommandTaskletIntegrationTests`, or should I create a separate `SystemCommandTaskletTests`?